src/analysis/inner_product_space/positive.lean

```diff
@@ -32,21 +42,329 @@ of requiring self adjointness in the definition.
 
 Positive operator
 -/
-
-open inner_product_space is_R_or_C continuous_linear_map
+open inner_product_space is_R_or_C
 open_locale inner_product complex_conjugate
+variables {𝕜 E F : Type*} [is_R_or_C 𝕜] [inner_product_space 𝕜 E] [inner_product_space 𝕜 F]
+local notation `⟪`x`, `y`⟫` := @inner 𝕜 _ _ x y
+
+/-- `U` is **pairwise orthogonal** to `V` if for all `u ∈ U` and for all `v ∈ V`
+we get `⟪u, v⟫ = 0` -/
+def submodule.pairwise_orthogonal (U V : submodule 𝕜 E) : Prop :=
+∀ u : U, ∀ v : V, ⟪(u : E), (v : E)⟫ = 0
```

Comment:
Am I right in saying that this condition is the same as `U <= Vᗮ`?

Reply:
Not sure. What if it's `Vᗮ <= U`?

Reply:
That's wrong; my spelling says "U fits within the space orthogonal to V", yours allows `V = U = \top`

Reply:
Why can't we have `V = U = \top`?

Reply:
Oh, right. So should we change the definition to say `U ≤ Vᗮ`?

Reply:
Yes, I think that's reasonable; but I also think that `pairwise` is probably not a useful description; maybe `U.is_orthogonal_to V` is a reasonable spelling; and could be given notation `U ⟂ V`.

Perhaps @jsm28 has some thoughts here?

Reply:
I like that. I like the notation `U ⟂ V`.

Reply:
It seems like a reasonable enough definition (with lemmas proving equivalence to various other forms of the definition), but probably not in `positive.lean` since it shouldn't depend on anything there. Rather, it might go in `basic.lean` (or in a file about orthogonal complements and families extracted from `basic.lean` in a separate PR, given how large `basic.lean` is).

Reply:
A new file looks reasonable to me.

Reply:
Cool, cool. I'll make a new PR for the new file soon then.

Reply:
Any news?

Reply:
@YaelDillies, I totally forgot about this! I did this last month, so I just need to make PRs. I'll do this by next week!

Reply:
I've split the file in #18706, and defined `submodule.is_ortho` in #18705

Reply:
This is now merged, and I've deleted `pairwise_orthogonal` from this PR. You might have some small CI errors to fix.